whois-api/src/main/java/net/ripe/db/whois/api/httpserver/WhoisDoSFilter.java

```diff
@@ -108,20 +108,16 @@ public void clearWhitelist() {
     @Override
     @ManagedOperation("adds an IP address that will not be rate limited")
     public boolean addWhitelistAddress(@Name("address") final String address) {
-        return addWhitelistAddress(address, true);
+        logWhiteList();
+        return addWhitelistAddressWithoutLog(address);
     }
 
-    private boolean addWhitelistAddress(final String address, final boolean log) {
-        boolean result;
+    private boolean addWhitelistAddressWithoutLog(final String address) {
```

Comment:
Do you need to name the method `WithoutLog` ? Is there another method that logs adding per address?

Reply:
yes:

    @Override
    @ManagedOperation("adds an IP address that will not be rate limited")
    public boolean addWhitelistAddress(@Name("address") final String address) {
        logWhiteList();
        return addWhitelistAddressWithoutLog(address);
    }

I refactor this because of duplicated log everywhere.